### PR TITLE
Fix for issue #509

### PR DESCRIPTION
**Pull request by bobroro**
as per @Mogztter to address Kroki extension missing attributes

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing documents:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

If you're adding or changing functionality:
- [ ] All tests are passing: https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/coder/building-and-running.html
- [ ] New/updated tests are included

If adding a **new feature**:
- [ ] Feature documentation is updated: https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/doc/users-guide/modules/ROOT/pages/features.adoc

**Other information:**

### CODE REVIEW

File: src/main/resources/kroki-extension.rb [hunk 1]
```diff
@@ -63,8 +64,16 @@ def process(processor, parent, attrs, diagram_type, diagram_text)
           'role' => role ? "#{role} kroki" : 'kroki',
           'target' => image_url,
           'alt' => target || 'diagram',
-          'title' => title
+          'title' => title,
```
Comment (Mogztter) [inline src/main/resources/kroki-extension.rb:67]: Apparently, the attribute `title` is not used so we should not set it:

```suggestion
```

Comment (Mogztter) [reply src/main/resources/kroki-extension.rb:67]: I just noticed that the `title` was already present... reading the code of Asciidoctor I think it's unused but without tests it's really hard to say. Maybe it's better to leave it for now.

**Comment by Mogztter** [comment]
Thanks @ahus1 for the quick merge :+1:

**Comment by ahus1** [comment]
@Mogztter / cc: @bobroro - no problem. I left the other issue open just in case you might need it. Please let me know if the other issue should be closed as well.

**Pull request merged**